src/main/resources/icon_associations.xml

```diff
@@ -141,7 +141,8 @@
         <regex name="EJS" pattern=".*\.(ejs)$" icon="/icons/files/ejs.svg"/>
         <regex name="Elixir" pattern=".*\.(ex|exs|elixir)$" icon="/icons/files/ex.svg"/>
         <regex name="Elm" pattern=".*\.elm$" icon="/icons/files/elm.svg"/>
-        <regex name="ESLint" pattern=".*\.(eslintrc|eslintignore)$" icon="/icons/files/eslint.svg"/>
+        <regex name="ESLint" pattern=".*\.eslintrc(\.js(on)?|\.y(a)?ml)?$" icon="/icons/files/eslint.svg"/>
+        <regex name="ESLint Ignore" pattern=".*\.eslintignore$" icon="/icons/files/eslint.svg"/>
```

Comment:
I don't want to add ignore files, this is done by another plugin, .ignore, which adds all the ignore extensions (.gitignore, .jshintignore and so on). Please remove it.